src/kvstore/wal/FileBasedWal.cpp

```diff
@@ -280,9 +280,17 @@ void FileBasedWal::closeCurrFile() {
         return;
     }
 
-    CHECK_EQ(fsync(currFd_), 0) << strerror(errno);
+    if (lastLogId_ > firstLogId_) {
+        if (::fsync(currFd_) == -1) {
+            LOG(WARNING) << "sync wal \"" << currInfo_->path()
+                         << "\" failed, error: " << strerror(errno);
+        }
+    }
     // Close the file
-    CHECK_EQ(close(currFd_), 0) << strerror(errno);
+    if (::close(currFd_) == -1) {
```

Comment:
Maybe we can discard CHECK of `utime` below as well. WDYT?

Reply:
ok, i will fix this.